### DISCUSSION TITLE
Donation Page

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#71] [Feature requests] [answered]
Let's identify things where donations are needed.

Web hosting fees
Public CA certificates
Code signing certificates

Suggest add option to send donations to support services and on-going care and feed.

**Comment by f1multiviewer** [comment, marked as answer]
Will be added on the next release!

**Reply by JJWatMyself** [reply]
Brilliant to see this live now.  Thank you.
